sdk/src/camera_96tof1.cpp

```diff
@@ -41,6 +41,12 @@ aditof::Status Camera96Tof1::initialize() {
 
     LOG(INFO) << "Camera initialized";
 
+		m_calibration.getIntrinsic(INTRINSIC, m_details.intrinsics.cameraMatrix);
+		m_calibration.getIntrinsic(DISTORTION_COEFFICIENTS,
+								   m_details.intrinsics.distCoeffs);
+		m_details.intrinsics.pixelWidth = 0.0056;
```

Comment:
Can you add a comment to specify where these hardcoded values come from?

Reply:
solved in commit https://github.com/analogdevicesinc/aditof_sdk/pull/203/commits/adba4901642e22b8626a1910e3c23271c2af6dc3